base/channels.jl

```diff
@@ -104,13 +157,12 @@ function Channel(func::Function; ctype=Any, csize=0, taskref=nothing)
     chnl = Channel{ctype}(csize)
     task = Task(() -> func(chnl))
     bind(chnl, task)
-    yield(task) # immediately start it
+    schedule(task)
```

Comment:
Should we deprecate `yield(task)` to `schedule(task)` on 0.7, immediately?

Reply:
I'd like to. And, I'm even more interested in deprecating `yieldto()`! But I haven't won that argument yet...